src/ducks/auth/sagas.ts

```diff
@@ -76,11 +81,21 @@ function* logoutSaga(): Generator<
   }
 }
 
+function* sendVerificationEmailSaga(): Generator<CallEffect<void>, void, void> {
+  try {
+    yield call(sendVerificationEmail);
+    console.log('Email sent successfully');
+  } catch (err) {
+    console.error(`Error to send email: ${err}`);
```

Comment:
Here we can track a positive response or an error, for this we can add 2 reducers